app/models/server/models/Users.js

```diff
@@ -273,7 +273,7 @@ export class Users extends Base {
 					},
 				},
 			},
-			...customFilter ? [customFilter] : [],
+			...customFilter ? [{ $match: customFilter }] : [],
```

Comment:
Are you sure this was required? TBH, I don't think this is required because `customFilters` are getting defined by [this](https://github.com/RocketChat/Rocket.Chat/blob/607d42db9bf42a7e25f3fa520d989efdbe5d8eca/ee/app/livechat-enterprise/server/hooks/applySimultaneousChatsRestrictions.ts#L5) callback and as you can see [here](https://github.com/RocketChat/Rocket.Chat/blob/607d42db9bf42a7e25f3fa520d989efdbe5d8eca/ee/app/livechat-enterprise/server/hooks/applySimultaneousChatsRestrictions.ts#L9) and [here](https://github.com/RocketChat/Rocket.Chat/blob/607d42db9bf42a7e25f3fa520d989efdbe5d8eca/ee/app/livechat-enterprise/server/hooks/applySimultaneousChatsRestrictions.ts#L32) both these return values apply a $match filter

Reply:
When I logged what we are receiving in this function, It was showing `{departmentId: 'string'}`.

FYI I don't have enterprise edition, so `/ee/app/livechat-enterprise/server/hooks/applySimultaneousChatsRestrictions.ts`  hook won't be called.

Maybe the issue is only with a non-enterprise server?

Reply:
Okay got it. I found out that this was happening because `livechat.applySimultaneousChatRestrictions` callback was simply returning the first param passed to it incase the callback was not found, which happens in the community edition. 

So I think the correct way to use this callback was to pass the params/constants in the second param and keep the first param empty/default. 

I took the liberty of raising a PR for the same https://github.com/RocketChat/Rocket.Chat/pull/22839

Reply:
Thanks @murtaza98 🙃